.github/workflows/test_build.yaml

```diff
@@ -3,15 +3,12 @@ on: push
 jobs:
   build-n-publish:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ['3.8', '3.9', '3.10']
     steps:
     - uses: actions/checkout@master
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python 3.8
       uses: actions/setup-python@v2
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: 3.8
```

Comment:
Why 3.8 and not 3.10? (or even 3.11)

Reply:
I think 3.8 as our minimum requirement works for the build test.